src/to-do.js

```diff
@@ -0,0 +1,127 @@
+import { check } from './validate.js';
+
+const InnerTodo = document.querySelector('.todo-body');
+let isEdit = false;
+let editId = null;
+
+export default class display {
+  static getTodo = () => {
+    let Todo;
+    if (localStorage.getItem('todo') == null) {
+      Todo = [];
+    } else {
+      Todo = JSON.parse(localStorage.getItem('todo'));
+    }
+    return Todo;
+  };
+
+  static addRemoveEvent = () => {
+    const trash = document.querySelectorAll('.trash');
+    trash.forEach((task, i) => {
+      task.addEventListener('click', (ev) => {
+        ev.preventDefault();
+        this.removeTodo(i);
+      });
+    });
+  };
+
+  static checkEvent = () => {
+    const toDos = display.getTodo();
+    const todoCheck = document.querySelectorAll('.to-do-check');
+    todoCheck.forEach((task, i) => {
+      const findTodo = toDos.find((item) => i === item.index);
+      task.addEventListener('change', (ev) => {
+        ev.preventDefault();
+        if (!findTodo.completed) {
+          ev.target.parentElement.style.textDecoration = 'line-through';
+        } else {
+          ev.target.parentElement.style.textDecoration = 'none';
+        }
+        check(toDos, i);
+      });
+    });
+  };
```

Comment:
I think avoiding using JavaScript to apply style directing to your HTML will make your code neater. You can use JavaScript to switch classes instead.

Reply:
Ok understood, Thanks